job-service-scheduled-executor/src/main/java/com/github/jobservice/scheduledexecutor/WorkerAction.java

```diff
@@ -48,6 +48,7 @@ public String toString() {
     private TaskDataEncodingEnum taskDataEncoding = null;
     private String taskPipe = null;
     private String targetPipe = null;
+    private String correlationId = null;
```

Comment:
Comment above that adding the correlation id to this class but not to the other generated class that is referenced is intentional and not just an oversight.